source/_integrations/enphase_envoy.markdown

```diff
@@ -685,6 +685,20 @@ When using Envoy Metered with <abbr title="current transformers">CT</abbr>
 
 - not all firmware versions report `Energy production today` and/or `Energy consumption today` correctly. Zero data and unexpected spikes have been reported. In this case, best use a utility meter with the `Lifetime energy production` or `Lifetime energy consumption` entity for daily reporting.
 - not all firmware versions report `Energy production last seven days` and/or `Energy consumption last seven days` correctly. Zero and unexpected values have been reported.
+- `Energy production today` has been reported not to reset to zero at the start of the day. Instead, it resets to a non-zero value that gradually increases over time. For daily reporting, it is recommended to use a utility meter with the `Lifetime energy production` entity. This issue has also been reported as starting suddenly overnight.
```

Comment:
```suggestion
- `Energy production today` has been reported not to reset to zero at the start of the day. Instead, it resets to a non-zero value that gradually increases over time. This issue has also been reported as starting suddenly overnight. For daily reporting, it is recommended to use a utility meter with the `Lifetime energy production` entity.
```
If "this issue" refers to the issue described in the first sentence, I would move it there. With the sentence at the end, it made me wonder if it refers to an issue related to using Lifetime energy production instead.